### DISCUSSION TITLE
Control style using css pseudo properties

### DISCUSSION THREAD

**Discussion by shiranZe** [chakra-ui/chakra-ui#5280] [Q&A]
Hi!

I have some `Box` elements rendered. 
I want to implement that if I click on one of them, 
the selected box element will have a certain style, let's say blue border.
How I can do it with chakra? I see that there's a `_selected` pseudo but how can I implement it on a `Box` element in a **dynamic** way? 

https://user-images.githubusercontent.com/40235967/147114860-5a6c8f59-e160-439b-9af7-2211df3d0eff.mov

**Comment by kk21** [comment]
```
<Box tabindex="0" _focus={{bgColor:"gray.500"}} bg='tomato' w='100%' p={4} color='white'>
  This is the Box
</Box>
```
Can also use tabindex="-1" to skip keyboard navigation.

**Reply by shiranZe** [reply]
Thank you!
But what if I have a state `Selected` that I have as a prop?
I would want the box element to be dynamically set as focused.
